packages/api/endpoints/granules.js

```diff
@@ -384,9 +434,12 @@ const handlePutAction = async (
       status: 'SUCCESS',
     });
   }
-  return res.boom.badRequest('Action is not supported. Choices are "applyWorkflow", "move", "reingest", "removeFromCmr" or specify no "action" to update an existing granule');
-};
+  return res.boom.badRequest(
+    'Action is not supported. Choices are "applyWorkflow", "move", "reingest", "removeFromCmr" or specify no "action" to update an existing granule'
+  );
+}
 
+/* eslint-disable complexity */ //TODO
```

Comment:
So there's this. I didn't want to do it here because this PR is already massive but I can either do a follow-up or make this change after the initial review.